### PR TITLE
Fix/ci/upgrade and lock clarinet core contract tests

### PR DESCRIPTION
**Pull request by netrome**

### CODE REVIEW

**Comment by jcnelson** [comment]
Thanks @netrome! I'll approve this for now, but what was the underlying problem?

**Review by jcnelson — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: "Execute core contract unit tests in Clarinet"
-        uses: docker://hirosystems/clarinet:latest
+        uses: docker://hirosystems/clarinet:1.1.0
```
Comment (lgalabru) [inline .github/workflows/ci.yml:89]: I think we need to use latest here.

Comment (lgalabru) [reply .github/workflows/ci.yml:89]: in the context of this repo, we could actually pin, the contract that clarinet is testing is a boot contract based on clarity 1, and we're not actively working on it. by pinning 1.1.0, we re pulling an unstable clarity 2 vm, but that's fine the only thing we need in this context is a stable clarity 1 vm.

Comment (netrome) [reply .github/workflows/ci.yml:89]: Great, thank you for the input! I prefer maintaining pinned versions to keep the CI more predictable.

**Pull request merged**